src/SFA.DAS.EmployerCommitmentsV2.Web/Startup/AspNetStartup.cs

```diff
@@ -20,6 +24,10 @@ public AspNetStartup(IConfiguration configuration)
 
         public void ConfigureServices(IServiceCollection services)
         {
+            var connectionStrings = new ConnectionStrings();
```

Comment:
I'm not sure this works; you need to `bind` the config, or, less elegantly, get it  through some means:
```
var redisConnectionString = _configuration.GetSection("SFA.DAS.EmployerCommitmentsV2").GetConnectionString("SharedRedis");
```